google-cloud-storage/samples/storage_set_bucket_public_iam.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def set_bucket_public_iam bucket_name:, role:, member:
```

Comment:
This isn't really a sample that makes a bucket public, it's a sample that allows the user to set a bucket's IAM to whatever they pass in. `role` and `member` should be removed from the method signature, and hardcoded within the sample, as in the [java sample](https://cloud.google.com/storage/docs/access-control/making-data-public#storage-make-bucket-public-java)

Reply:
Done, thanks for the link.